### ISSUE TITLE
Challenges - Empty States for Ongoing/History

### ISSUE THREAD

**Issue by D3Portillo** [Arena-Mayhem/frontend#14]
## TL;DR

Suggest an action to users when not connected or no data found for their challenges

## Component Story

- As a user in the "ONGOING" section or "HISTORY" section that is NOT connected to the platform I want to see a widget that asks me to connect my wallet
- As a user in the "ONGOING" section or "HISTORY" section that IS connected to the platform and doesn't have any challenges data I want to see a widget that asks me to create challenge

## References

Not connected:
![Image](https://github.com/user-attachments/assets/febe729a-98ed-41c8-a277-405e64a2a92a)

No data:
![Image](https://github.com/user-attachments/assets/0697d599-7164-4a28-a171-c412e66632ed)

## Resources

- Figma View: https://www.figma.com/design/csb1dd9yjNAg4sWTmVEZww/Arena-Mayhem?node-id=27-2518&t=htvhpzMvSyVVJfRj-0

---

> [!TIP]
> A nice to read. This is a brief explanation of the types of empty states https://carbondesignsystem.com/patterns/empty-states-pattern/#no-data-empty-states

**Comment by cyanidev** [comment]
![image](https://github.com/user-attachments/assets/4a840150-cb21-46e2-91a6-93ac4784731a)
![image](https://github.com/user-attachments/assets/00b93ff0-02b2-4e5e-abc5-f53485721f76)
![image](https://github.com/user-attachments/assets/8eccef1b-a733-4997-bcd3-8bd4fb2342a8)
![image](https://github.com/user-attachments/assets/bbcf678b-e0d5-4c3b-9b6b-2cf6b251ca7f)
![image](https://github.com/user-attachments/assets/69e43d15-a90c-44c1-b921-901ea1c53dff)
